### PR TITLE
UML-1314 - Upgrade OS packages

### PR DESCRIPTION
**Pull request by andrewpearce-digital**
# Purpose

Check the status of the image after OS package upgrades

Fixes UML-1314

## Approach

run an apt-get update and upgrade to see the end result of image scanning (This did not fix the issue)

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

**Review by SeemaMenon — DISMISSED** [review]

**Comment by andrewpearce-digital** [comment]
PDF download looks ok
image

**Review by GemTay — APPROVED** [review]

**Comment by andrewpearce-digital** [comment]
Simple test for pdf link added
image

**Review by GemTay — APPROVED** [review]
Nice!

File: tests/smoke/context/CommonContext.php [hunk 1]
```diff
@@ -217,4 +217,12 @@ public function iReceiveHeadersThatBlockExternalIndexing()
         assertContains('nofollow', $xrobotstag);
         assertContains('noindex', $xrobotstag);
     }
+
+    /**
+     * @Then I am given a PDF file of the summary
+     */
+    public function IAmGivenAPDFFileOfTheSummary(): void
+    {
+        $this->ui->assertResponseStatus(StatusCodeInterface::STATUS_OK);
```
Comment (cooperaj) [inline tests/smoke/context/CommonContext.php:226]: Might be worth visiting this further to do some other checks (has right content type, is pdf etc)

**Pull request merged**